src/pages/office/components/candidate-description/candidate-description.ts

```diff
@@ -17,19 +26,27 @@ export class CandidateDescriptionComponent implements OnChanges {
   candidateDescriptionChange = new EventEmitter<string>();
 
   private formControl: FormControl;
+  private fieldName: string = 'candidateDescription';
```

Comment:
this can turned into a constant by using the `static readonly` configuration

There are a few other cases in this PR where this same comment apply

Reply:
changed all components